lib/galaxy/dependencies/dev-requirements.txt

```diff
@@ -112,95 +116,98 @@ openstacksdk==0.52.0; python_version >= "3.6"
-pyjwt==1.7.1; python_version >= "3.6"
+pyjwt==2.0.1; python_version >= "3.6"
 pykwalify==1.8.0
 pynacl==1.4.0; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.4.0"
 pyparsing==2.4.7; (python_version >= "2.6" and python_full_version < "3.0.0") or (python_full_version >= "3.3.0")
-pyperclip==1.8.1; python_version >= "3.6"
-pyreadline==2.1; python_version >= "3.6" and python_full_version < "3.0.0" and python_version < "4" and sys_platform == "win32" or python_version >= "3.6" and python_version < "4" and python_full_version >= "3.5.0" and sys_platform == "win32"
+pyperclip==1.8.2; python_version >= "3.6"
+pyreadline3==3.3; sys_platform == "win32" and python_version >= "3.8"
+pyreadline==2.1; python_version >= "3.6" and python_full_version < "3.0.0" and python_version < "3.8" and sys_platform == "win32" or python_version >= "3.6" and python_version < "3.8" and python_full_version >= "3.5.0" and sys_platform == "win32"
+pyrsistent==0.17.3; python_version >= "3.5"
 pysam==0.16.0.1
-pytest-asyncio==0.14.0
+pytest-asyncio==0.14.0; python_version >= "3.5"
 pytest-cov==2.11.1; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.5.0")
 pytest-html==3.1.1; python_version >= "3.6"
+pytest-json-report==1.2.4
 pytest-metadata==1.11.0; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6"
 pytest-mock==3.5.1; python_version >= "3.5"
-pytest-postgresql==2.5.2; python_version >= "3.6"
+pytest-postgresql==2.6.1; python_version >= "3.6"
 pytest-pythonpath==0.7.3
 pytest-shard==0.1.2; python_version >= "3.6"
-pytest==6.2.1; python_version >= "3.6"
+pytest==6.2.2; python_version >= "3.6"
 python-dateutil==2.8.1; python_version >= "3.6" and python_full_version < "3.0.0" and python_version < "4" or python_version >= "3.6" and python_version < "4" and python_full_version >= "3.3.0"
-python-irodsclient==0.8.5
-pytest-json-report==1.2.1
+python-irodsclient==0.8.6
 python-jose==3.2.0
 python-keystoneclient==4.1.1; python_version >= "3.6"
 python-neutronclient==7.2.1; python_version >= "3.6"
 python-novaclient==17.2.1; python_version >= "3.6"
 python-swiftclient==3.10.1
 python3-openid==3.2.0; python_version >= "3.0"
-pytz==2020.5; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version < "4" and python_version >= "3.6"
+pytz==2021.1; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version < "4" and python_version >= "3.6"
 pyuwsgi==2.0.19.1.post0
 pyyaml==5.4.1; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.6.0")
 rdflib-jsonld==0.5.0; python_version >= "3.6" and python_version < "4"
 rdflib==4.2.2; python_version >= "3.6" and python_version < "4"
 recommonmark==0.7.1
-refgenconf==0.9.3
+refgenconf==0.10.0
```

Comment:
Given https://github.com/galaxyproject/galaxy/issues/11601 , we should pin refgenconf in pyproject.toml

Reply:
Done in https://github.com/galaxyproject/galaxy/pull/11627